main.py

```diff
@@ -52,14 +72,17 @@ def clear():
 if "hyper-v" in settings and settings.get("hyper-v").get("enabled", True):
     hypervisors.append("hyper-v")
     settings["hyper-v"]["enabled"] = True
+if "virtualbox" in settings and settings.get("virtualbox").get("enabled", True):
+    hypervisors.append("virtualbox")
+    settings["virtualbox"]["enabled"] = True
 if hypervisors == []:
     if Path("hypervisors.txt").is_file():
         # Client ID found in legacy file
         hypervisors = Path("hypervisors.txt").read_text()
         hypervisors = hypervisors.casefold().split("\n")
     else:
-        hypervisors = ["vmware", "hyper-v"]
-        settings.update({'vmware': {'enabled': True}, 'hyper-v': {'enabled': True}})
+        hypervisors = ["vmware", "hyper-v", "virtualbox"]
+        settings.update({'vmware': {'enabled': True}, 'hyper-v': {'enabled': True}, 'virtualbox': {'enabled': True}})

```

Comment:
Not sure about these 2 lines, never worked with hyper-v

Reply:
Looks fine